bte/asgi.py

```diff
@@ -0,0 +1,16 @@
+"""
```

Comment:
Is this file used? I saw a wsgi file too?

Reply:
yes ... wsgi file is called from the docket entrypoint script.

Reply:
asgi ... not so much and was removed